README.md

```diff
@@ -83,12 +83,33 @@ On the file [`helmfile-common.yaml`](./helmfile-common.yaml) set your Helm Regis
 
 4. Declare the blockchain networks you want to connect Orchestrate to, separate them by an space.
 
-```
+```yaml
 chainRegistry:
   init:'{"name":"<ChainName1>","tenantID":"<tenatID1>", "urls":["<list item 1A>","<list item 1B>"]} {"name":"<ChainName2>","tenantID":"<tenatID2>", "urls":["<list item 2A>","<list item 2B>"]}}'
 ```
 
-#### Only when using Hashicorp Vault in AWS
+5. Add the environment '<KubernetesNameSpace>' into the file  `helmfile-common.yaml` 
```

Comment:
There should be an empty line after the bullet point sentence. I cant add it myself, sorry for that.